iceoryx_hoofs/include/iceoryx_hoofs/internal/concurrent/lockfree_queue/index_queue.inl

```diff
@@ -226,9 +232,9 @@ bool IndexQueue<Capacity, ValueType>::popIfFull(ValueType& index) noexcept
 }
 
 template <uint64_t Capacity, typename ValueType>
-bool IndexQueue<Capacity, ValueType>::popIfSizeIsAtLeast(const uint64_t requiredSize, ValueType& index) noexcept
+bool IndexQueue<Capacity, ValueType>::popIfSizeIsAtLeast(const uint64_t minSize, ValueType& index) noexcept
```

Comment:
Could you please also adjust the name in the definition in the corresponding hpp

Reply:
Already the case, they were out of sync and that is why it was changed here.